Emby.Server.Implementations/ConfigurationOptions.cs

```diff
@@ -18,7 +18,7 @@ public static class ConfigurationOptions
         {
             { HostWebClientKey, bool.TrueString },
             { HttpListenerHost.DefaultRedirectKey, "web/index.html" },
-            { InstallationManager.PluginManifestUrlKey, "https://repo.jellyfin.org/releases/plugin/manifest.json" },
+            { InstallationManager.PluginManifestUrlKey, "https://repo.jellyfin.org/releases/plugin/manifest-stable.json" },
```

Comment:
I would rather we keep the default... not a hard preference, though. But renaming it would break existing installs, so why?..

Reply:
This will require changes to the manifest formatting, and we can't break old versions by updating the file in place, which is why we need a new URL for the latest changes. When I originally made this pull request the value was hardcoded and would have updated fine, but this is indeed a problem now.

Reply:
So this makes old manifest a legacy one, right?

Reply:
Yes, that's correct.